shallow_backup/printing.py

```diff
@@ -83,23 +84,31 @@ def print_section_header(title, color):
 
 
 def print_pkg_mgr_backup(mgr):
-	print("{}Backing up {}{}{}{}{} packages list...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr, Fore.BLUE,
-	                                                          Style.NORMAL, Style.RESET_ALL))
+	print("{}Backing up {}{}{}{}{} packages list...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr,
+															  Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
 
 
 def print_pkg_mgr_reinstall(mgr):
-	print("{}Reinstalling {}{}{}{}{}...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW, mgr, Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
+	print("{}Reinstalling {}{}{}{}{}...{}".format(Fore.BLUE, Style.BRIGHT, Fore.YELLOW,
+												  mgr, Fore.BLUE, Style.NORMAL, Style.RESET_ALL))
 
 
-def prompt_yes_no(message, color):
+def print_shell_cmd_error(cmd):
+	print("{}$ {}".format(print_path_red('An error occurred while running: '), cmd))
+
+
+def prompt_yes_no(message, color, invert=False):
 	"""
 	Print question and return True or False depending on user selection from list.
 	"""
 	questions = [inquirer.List('choice',
 	                           message=color + Style.BRIGHT + message + Fore.BLUE,
-	                           choices=[' Yes', ' No'],
+	                           choices=(' No', ' Yes') if invert else (' Yes', ' No')
```

Comment:
Should these be lists? Honestly not sure if it matters or not, but let's change it back to what I know works.

```suggestion
	                           choices=[' No', ' Yes'] if invert else [' Yes', ' No']
```

Reply:
I liked the tuples for its immutable indication but I'll change it back

Reply:
If it works, I'm cool with it. I just haven't tested the most recent commits on this PR.